firestore/src/main/firestore_main.h

```diff
@@ -48,6 +48,7 @@ class Firestore;
 class ListenerRegistrationInternal;
 class Transaction;
 class WriteBatch;
+class Settings;
```

Comment:
nit: Put the forward declaration of `Settings` in its correct alphabetical ordering among the other forward declarations.